Rock/Web/UI/Controls/AttributeEditor.cs

```diff
@@ -1486,6 +1488,24 @@ protected void CreateFieldTypeDetailControls( int? fieldTypeId, bool recreate =
         }
 
         /// <summary>
+        /// Set the Subtitle of modal dialog
+        /// </summary>
+        /// <param name="attribute">The attribute.</param>
+        protected void SetSubTitleOnModal( Model.Attribute attribute )
+        {
+            Control parent = this.Parent;
+            while ( parent.GetType() != typeof( ModalDialog ) )
+            {
+                parent = parent.Parent;
+            }
+            if ( parent.GetType() == typeof( ModalDialog ) )
+            {
+                ModalDialog modalDialog = ( ModalDialog ) parent;
+                modalDialog.SubTitle = string.Format( "Id: {0}", attribute.Id );
+            }
```

Comment:
Might only set subtitle too if it's blank (in case a block is using subtitle already for something else).